cmd/juju/status/status_test.go

```diff
@@ -1814,7 +1826,7 @@ var statusTests = []testCase{
 
 		// scoped on wordpress/0
 		scopedExpect{
-			"subordinates scoped on logging",
```

Comment:
This was a copy/paste error from a previous test. Updated the comment of the test to match the args passed to status, and the comment just before the 'scopedExpect'